### PR TITLE
REF rerendering w/ actions

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Pushed the branch to main repo
* [x] CI passed on the branch

### CODE REVIEW

File: conda_forge_webservices/commands.py [hunk 1]
```diff
@@ -88,17 +110,19 @@ def pr_detailed_comment(org_name, repo_name, pr_owner, pr_repo, pr_branch, pr_nu
...
     with tmp_directory() as tmp_dir:
-        print(tmp_dir, repo_name)
         feedstock_dir = os.path.join(tmp_dir, repo_name)
         repo_url = "https://{}@github.com/{}/{}.git".format(
             os.environ['GH_TOKEN'], pr_owner, pr_repo)
         repo = Repo.clone_from(repo_url, feedstock_dir, branch=pr_branch)
 
+        gh = github.Github(os.environ['GH_TOKEN'])
+        gh_repo = gh.get_repo("{}/{}".format(org_name, repo_name))
+        pull = gh_repo.get_pull(int(pr_num))
```
Comment (isuruf) [inline conda_forge_webservices/commands.py]: This block was inside the if statements, to avoid GH API calls

Comment (beckermr) [reply conda_forge_webservices/commands.py]: Got it. I’ll put it down far as possible.

Comment (beckermr) [reply conda_forge_webservices/commands.py]: moved

**Comment by beckermr** [comment]
@isuruf @conda-forge/core any other comments here?

**Pull request merged**